subsys/bluetooth/host/Kconfig

```diff
@@ -492,6 +492,18 @@ config BT_BACKGROUND_SCAN_WINDOW
 	int "Scan window used for background scanning in 0.625 ms units"
 	default 18
 	range 4 16384
+
+if BT_EXT_ADV
+config BT_EXT_SCAN_BUF_SIZE
+	int "Maximum advertisement report size"
+	range 1 1650
```

Comment:
Does this value only include the `Data` of the reports, or the size of the report itself? 

If so, then I guess then range could be `range 1 1650` as advertising report data may be empty (0 length).

Reply:
It includes the data itself. That is, you can receive 0 or 1 bytes in each advertising report. 

Did you intend to suggest `range 0 1650`? Using 1 as a minimum value simplifies the implementation as we don't have to consider the possibility of creating a 0-length buffer.

Reply:
> Did you intend to suggest range 0 1650?

Indeed. This could be a later optimization; if set to 0 we could probably remove quite a bit of code, but it's fine to postpone that for me. It could of course also be argued that if set to 1 we could omit the code for handling segmentation.